test/src/test/java/org/corfudb/integration/CorfuReplicationReconfigurationIT.java

```diff
@@ -87,7 +87,11 @@ public class CorfuReplicationReconfigurationIT extends LogReplicationAbstractIT
     private Table<StringKey, IntValueTag, Metadata> noisyMap;
 
     public CorfuReplicationReconfigurationIT(Pair<String, ExampleSchemas.ClusterUuidMsg> pluginAndTopologyType) {
-        this.pluginConfigFilePath = pluginAndTopologyType.getKey();
+        if (pluginAndTopologyType.getKey().equals("GRPC")) {
+            System.setProperty("transport", "GRPC");
+        } else {
+            System.setProperty("transport", "NETTY");
```

Comment:
Any reason why the netty type is kept in this test only?  All other tests are using the GRPC plugin only as netty is not available right now.